### PR TITLE
Added Clickie source

### PR DESCRIPTION
**Pull request by peybol**

### CODE REVIEW

File: sources/clickie.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
+  "id": "CLICKIE",
+  "name": "Clickie",
+  "categories": ["IOT", "TIME_SERIES"],
+  "organization": "Clickie SPA",
```
Comment (pfrisella) [inline sources/clickie.json:5]: not a valid org. See https://github.com/googledatastudio/ds-data-registry#add-an-organization

**Review by pfrisella — CHANGES_REQUESTED** [review]

File: sources/clickie.json (hunk 1 above)
Comment (peybol) [reply sources/clickie.json]: fixed, thanks.

File: organizations/clickie.json [hunk 2]
```diff
@@ -0,0 +1,6 @@
+{
+  "id": "CLICKIE_SPA",
```
Comment (pfrisella) [inline organizations/clickie.json:2]: ID and filename must match. filename should be clickie_spa.json

**Review by pfrisella — CHANGES_REQUESTED** [review]

File: organizations/clickie.json (hunk 2 above)
Comment (peybol) [reply organizations/clickie.json]: Sorry, done. Thanks.

**Review by pfrisella — APPROVED** [review]
LGTM. Thanks for adding!

**Pull request merged**